### PR TITLE
feat: arbitrum nova

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Arbitrum Nova standard support

- Send, Receive, Balance History and Transaction History
- Only support for coingecko asset list currently
- No swap or nft support

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk

Low

## Testing

- Send/Receive (shapeshift wallet, metamask, wallet connect)
- Validate transaction history
- Validate balance charts

Note: there is currently a very small supported asset list provided by coingecko. Follow up PR to add placeholder assets to allow seeing tx history and crypto balance (no market data) for missing assets.

### Engineering

:point_up:

### Operations

:point_up:

### CODE REVIEW

File: src/state/migrations/index.ts [hunk 1]
```diff
@@ -13,4 +13,5 @@ export const migrations = {
   6: clearAssets,
   7: clearPortfolio,
   8: clearOpportunities,
+  9: clearAssets,
```
Comment (kaladinlight) [inline src/state/migrations/index.ts:16]: clear assets for newly generated asset data

File: src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts [hunk 2]
```diff
@@ -67,6 +68,7 @@ export const walletSupportsChain: UseWalletSupportsChain = ({
     case gnosisChainId:
       return supportsGnosis(wallet)
     case arbitrumChainId:
+    case arbitrumNovaChainId:
       return supportsArbitrum(wallet)
```
Comment (kaladinlight) [inline src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts:72]: should probably add `supportsArbitrumNova` just to be extra cautious?

Comment (0xApotheosis) [reply src/hooks/useWalletSupportsChain/useWalletSupportsChain.ts:72]: We should, though fine as a follow-up.

**Review by 0xApotheosis — APPROVED** [review]
Very tidy big kev, everything is looking as it should.

Happy to get this in, as still feature flagged.

Screenshot 2023-11-06 at 9 46 26 am

Screenshot 2023-11-06 at 9 46 50 am

**Pull request merged**